### PR TITLE
Feature/rommel caption memes

### PR DESCRIPTION
**Pull request by RommelTJ**
My memes. I had to update the gitignore because I’m using the IntelliJ.
Approved !

### CODE REVIEW

**Review by abadwal — APPROVED** [review]
looks cool

File: _posts/0000-01-02-RommelTJ.md [hunk 1]
```diff
@@ -3,7 +3,9 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
+If actions are stronger than words,  
+
 ![philosoraptor](https://cloud.githubusercontent.com/assets/16547949/25401001/66d677c6-29c2-11e7-9b12-263e036c236e.jpg)
 {: .center}
 
-Caption here
+why is the pen mightier than the sword?
```
Comment (githubteacher) [inline _posts/0000-01-02-RommelTJ.md:11]: Who says it is?

File: _posts/0000-01-07-RommelTJ.md [hunk 2]
```diff
@@ -3,7 +3,9 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
+Brace yourselves
+
 ![winter-is-coming](https://cloud.githubusercontent.com/assets/16547949/25401218/203cc85a-29c3-11e7-8bbf-8c7d4a6b81b2.jpg)
 {: .center}
 
-Caption here
+Github is coming!
```
Comment (githubteacher) [inline _posts/0000-01-07-RommelTJ.md]: YES! I might use "GitHub" though.

**Review by githubteacher — COMMENTED** [review]
This is great! Left a recommendation, but if you want to merge as is, that's fine, too.

File: _posts/0000-01-07-RommelTJ.md (hunk 2 above)
Comment (RommelTJ) [reply _posts/0000-01-07-RommelTJ.md]: Fixed.

File: _posts/0000-01-02-RommelTJ.md (hunk 1 above)
Comment (RommelTJ) [reply _posts/0000-01-02-RommelTJ.md:11]: My English teacher.

**Pull request merged**